sources/__tests__/getRole.js

```diff
@@ -94,7 +94,7 @@ const cases = [
 	["input type=checkbox", "checkbox", createElementFactory("input", {type: "checkbox"})],
 	["input type=color", null, createElementFactory("input", {type: "color"})],
 	["input type=date", null, createElementFactory("input", {type: "date"})],
-	["input type=datetime", null, createElementFactory("input", {type: "datetime"})],
+	["input type=datetime-local", null, createElementFactory("input", {type: "datetime-local"})],
```

Comment:
[`datetime` is obsolete](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime)